Backprojection.ipynb

```diff
@@ -121,52 +121,52 @@
     "                #xindex = x1 + 15.5\n",
     "                #yindex = y1 + 15.5\n",
     "                if abs(d) < 0.5:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] + Projection[n, projectionindex]\n",
-    "                if abs(d) >= 0.5 and abs(d) < 1:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] + .1 * Projection[n, projectionindex]\n",
-    "                if abs(d) >= 1 and abs(d) < 1.5:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] - .5 * Projection[n, projectionindex]\n",
-    "                if abs(d) > 1.5 and abs(d) < 2:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] - .4 * Projection[n, projectionindex]\n",
+    "                    set_backprojection(1)\n",
+    "                if 0.5 <= abs(d) < 1:\n",
+    "                    set_backprojection(.1)\n",
+    "                if 1 <= abs(d) < 1.5:\n",
+    "                    set_backprojection(-.5)\n",
+    "                if 1.5 < abs(d) < 2:\n",
+    "                    set_backprojection(-.4)\n",
     "        for n in range(18, 34):\n",
     "            for projectionindex in range (0, 30):\n",
     "                s = projectionindex-15.5\n",
     "                dd = -(yindex-15.5) * math.cos(n*delta_theta*math.pi/180) + s + math.sin(n*delta_theta*math.pi/180) * (xindex-15.5)\n",
     "                #xindex = x1 + 15.5\n",
     "                #yindex = y1 + 15.5\n",
     "                if abs(dd) < 0.5:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] + Projection[n, projectionindex]\n",
-    "                if abs(dd) >= 0.5 and abs(dd) < 1:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] + .1 * Projection[n, projectionindex]\n",
-    "                if abs(dd) >= 1 and abs(dd) < 1.5:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] - .5 * Projection[n, projectionindex]\n",
-    "                if abs(dd) > 1.5 and abs(dd) < 2:\n",
-    "                    Backprojection[xindex, yindex] = Backprojection[xindex, yindex] - .4 * Projection[n, projectionindex]"
+    "                    set_backprojection(1)\n",
+    "                if 0.5 <= abs(dd) < 1:\n",
+    "                    set_backprojection(.1)\n",
+    "                if 1 <= abs(dd) < 1.5:\n",
+    "                    set_backprojection(-.5)\n",
+    "                if 1.5 < abs(dd) < 2:\n",
+    "                    set_backprojection(-.4)"
```

Comment:
From line 110 to here is the only area where I made actual changes to the code.

I guess the rest of the changes are from running it in the notebook.